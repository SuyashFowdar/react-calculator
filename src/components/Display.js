import React from 'react';
import PropTypes from 'prop-types';

const Display = props => {
  const { result } = props;

  return (
    <div>
      <div>{result}</div>
    </div>
  );
};

Display.propTypes = {
  result: PropTypes.string.isRequired,
};

export default Display;
