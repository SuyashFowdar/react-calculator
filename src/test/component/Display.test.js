import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/Display';

it('renders correctly', () => {
  const tree = renderer
    .create(<Display result="1" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
