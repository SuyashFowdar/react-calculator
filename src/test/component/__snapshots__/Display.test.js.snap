// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="display row cross-center main-end"
>
  <div>
    1
  </div>
</div>
`;
